src/manifolds/ProbabilitySimplex.jl

```diff
@@ -421,16 +452,119 @@ function riemannian_gradient!(M::ProbabilitySimplex, X, p, Y; kwargs...)
     return X
 end
 
+@doc raw"""
+    riemann_tensor(::ProbabilitySimplex, p, X, Y, Z)
+
+Compute the Riemann tensor ``R(X,Y)Z`` at point `p` on [`ProbabilitySimplex`](@ref) `M`.
+It is computed using [`RealProbabilityAmplitudes`](@ref) isometry with positive orthant of
+a sphere.
+"""
+riemann_tensor(::ProbabilitySimplex, p, X, Y, Z)
+
+function riemann_tensor!(::ProbabilitySimplex{N}, Xresult, p, X, Y, Z) where {N}
+    ME = RealProbabilityAmplitudes(N)
+    pe = embed(ME, p)
+    Xrs = riemann_tensor(Sphere(N), pe, embed(ME, p, X), embed(ME, p, Y), embed(ME, p, Z))
+    project!(ME, Xresult, pe, Xrs)
+    return Xresult
+end
+
 function Base.show(io::IO, ::ProbabilitySimplex{n,boundary}) where {n,boundary}
-    return print(io, "ProbabilitySimplex($(n); boundary=$boundary)")
+    return print(io, "ProbabilitySimplex($(n); boundary=:$boundary)")
 end
 
 @doc raw"""
     zero_vector(M::ProbabilitySimplex, p)
 
-returns the zero tangent vector in the tangent space of the point `p`  from the
+Return the zero tangent vector in the tangent space of the point `p`  from the
 [`ProbabilitySimplex`](@ref) `M`, i.e. its representation by the zero vector in the embedding.
 """
 zero_vector(::ProbabilitySimplex, ::Any)
 
 zero_vector!(::ProbabilitySimplex, X, p) = fill!(X, 0)
+
+@doc raw"""
+    RealProbabilityAmplitudes
+
+An explicit isometric embedding of interior of [`ProbabilitySimplex`] in
+positive orthant of the [`Sphere`]. Some properties extend to the boundary but not all.
+
+This embedding isometrically maps the Fisher-Rao metric on the open probability simplex to
+the sphere of radius 1 with Euclidean metric. More details can be found in Section 2.2
+of [^AyJostLeSchwachhöfer2017].
```

Comment:
Just wanted to improve my understanding by asking a question about how this works.

It looks like you're missing the factor of 2 in the differential (of [f](https://github.com/JuliaManifolds/Manifolds.jl/issues/606#issuecomment-1553249494) and [finv](https://github.com/JuliaManifolds/Manifolds.jl/issues/606#issuecomment-1546866436)) when I look at
`embed!(::RealProbabilityAmplitudes, Y, p, X)` and `project!(::RealProbabilityAmplitudes, Y, p, X)` (which I think are supposed to serve this purpose), so I'm not sure this embedding is still isometric?

I only tried the parallel transport with one set of points and it looked correct though so I'm not sure there's a problem in this construction.

Reply:
The challenge here might be, that we (contrary to theory, but in favour of our iplementation) embed into the 1-sphere and not the 2-sphere. This is done in order to reuse the code we have for the 1-sphere; but our sphere does not have a way do incorporate a radius.

Reply:
The difference is indeed that those `f` and `finv` functions embed in a sphere of radius 2, while the embedding in this PR is in sphere of radius 1. That's why there is no factor of 2.

Reply:
I see, thanks both for clarifying and for working on these new functions 👍